news/2024/2024-07-29-the-followpoint-flaro-the-queen-of-speed.md

```diff
@@ -0,0 +1,205 @@
+
+How did you get introduced to osu!, and what was your first impression of it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>
+
+Shortly before my 11th birthday, I was introduced to the game by a friend of mine that I met through discord, at the time, the game didn't really captivate me that much and I stopped playing up until I turned 12, which is when I would start to play the game very actively and thought the game was very fun!
+
+I actually didn't even know what 'pp' was until I got my first 200, which is when I would start farming for ranks competitively.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+What made you come up with the name "Flaro", and is there any special meaning behind it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>
+
+Although the name commonly gets misconstrued as a spin on the pokemon 'Flareon', the name Flaro actually just came to me while I was laying in bed one night and I liked it more than my previous name.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+Out of all of the skillsets in the game, what made you want to prioritize speed?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10323184" style="background-image: url('/wiki/shared/news/2024-07-29-the-followpoint-flaro-the-queen-of-speed/avatar-Flaro.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10323184">Flaro</a></p>
+
+Funnily enough, I was exclusively a low bpm flow aim and jump player up until only mid 2022, which is when I would start to grind speed, going for a score on the one and only Lionheart map. It was when I skipped 800 in a Walmart whilst wearing a Megumin cosplay (true story!) that I decided to fully commit to getting better at speed and flow aim.
```

Comment:
```suggestion
Funnily enough, I was exclusively a low BPM flow aim and jump player up until only mid-2022, which is when I would start to grind speed, going for a score on the one and only [Lionheart](https://osu.ppy.sh/beatmapsets/1390922) map. It was when I skipped 800 in a Walmart whilst wearing a Megumin cosplay (true story!) that I decided to fully commit to getting better at speed and flow aim.
```

Reply:
was there not a follow up question to the megumin cosplay in a walmart? i feel like thats something that will peak players' interest but its completely left alone?

Reply:
Asked Flaro a follow up question to this, did the same with the "It says in your Discord bio that you are a Python Developer, how did you get into coding in Python, and what do you like most about it?" question as well.

Currently going to add the suggested changes for now with the other consistency changes I have and add the follow up questions when I get a response.